sysml.library/Kernel Library/Occurrences.kerml

```diff
@@ -161,19 +167,19 @@ package Occurrences {
 	}
 	
```

Comment:
I also forgot to update PortionOf comment and multiplicity in the same way TimeSliceOf is below:
/** PortionOf asserts one occurrence is a portion of another, including at least itself. */
end portion feature 'portion': Occurrence[1..*] redefines Occurrence::portions, BinaryLink::target;

Reply:
Updated.